cartographer/transform/transform_interpolation_buffer.h

```diff
@@ -27,18 +28,28 @@
 namespace cartographer {
 namespace transform {
 
+constexpr size_t kUnlimitedBufferSize = std::numeric_limits<size_t>::max();
+
 // A time-ordered buffer of transforms that supports interpolated lookups.
+// Unless explicitly set, the buffer size is unlimited.
 class TransformInterpolationBuffer {
  public:
   TransformInterpolationBuffer() = default;
   explicit TransformInterpolationBuffer(
       const mapping::proto::Trajectory& trajectory);
 
+  // Sets the transform buffer size limit and removes old transforms
+  // if it is exceeded.
+  void SetSizeLimit(const size_t buffer_size_limit);
```

Comment:
Nit: the `const` should be removed as it has no meaning here. It is only useful at the implementation.

Reply:
done